### ISSUE TITLE
JavaScript Support

### ISSUE THREAD

**Issue by prasannavl** [microsoft/bond#107]
Is there any way to consume Bond data from Javascript?

**Comment by sapek** [comment]
You can use [Simple](https://microsoft.github.io/bond/manual/bond_cpp.html#simple-json) [JSON](https://microsoft.github.io/bond/manual/bond_cs.html#json) protocol.

**Comment by prasannavl** [comment]
Hi, I was wondering if there's a way to decode the binary protocols. Internal communication where only efficiency matters, but where javascript tools pick up. I'm looking for a way to avoid dynamic wire-protocols, and retain the same binary.

**Comment by sapek** [comment]
No. It is a possibility for the future but today we don't have OSS JavaScript mappings for Bond.

**Comment by chwarr** [comment]
As of September 2020 and for the foreseeable future, no new languages will be added to Bond by Microsoft.

**Comment by prasannavl** [comment]
Thanks. Let me go ahead and close this.

**Comment by chwarr** [comment]
We can leave this issue open. JavaScript support is something that still makes sense for Bond.

It just isn't something that Microsoft is able to work on right now, nor do we know when we'll be able to work on it.

Someone else is welcome to contribute JavaScript support.
